lib/collectionspace/tools/date_parser.rb

```diff
@@ -40,7 +42,7 @@ def self.fields_for(date)
 
       def self.parse(date_string, end_date_string = nil)
         begin
-          unprocessed_date_string = date_string
+          unprocessed_date_string = date_string.clone
```

Comment:
Keep the unprocessed_date_string safe in case we need it later - otherwise it's a reference to date_string?

Reply:
Not in Ruby, in this case it's a copy of the value (actually a reference to the "value" under the hood I believe, not to the variable itself):

```
[1] pry(main)> original = 'original_string'
=> "original_string"
[2] pry(main)> unprocessed = original
=> "original_string"
[3] pry(main)> original = 'modified'
=> "modified"
[4] pry(main)> unprocessed
=> "original_string"
[5] pry(main)> unprocessed = 'something else'
=> "something else"
[6] pry(main)> original
=> "modified"
```

But the change is fine.

Reply:
Huh. Clearly I'm confused, but I thought I read up on this/verified it a few months ago. WILL NOT go down rabbithole of trying to figure out why I thought this was necessary... not now anyway...